samples/bookinfo/src/productpage/productpage.py

```diff
@@ -70,6 +71,13 @@
     "children": []
 }
 
+# Layotto Detail
+layottoDetails = {
```

Comment:
How about naming it as `previews` instead of `layotto xxxx`?  It's better that we make the code irrelevant with `layotto`